README.md

```diff
@@ -725,6 +725,20 @@ For older versions please use the type definitions from [DefinitelyTyped](https:
 npm install --save-dev @types/node-fetch
 ```
 
+## ECMAScript modules -- development and tests
```

Comment:
This should go in a `contributing.md`.

Reply:
@Richienb I've started a CONTRIBUTING.md file and moved this section there